src/main/java/org/sonar/plugins/stash/issue/collector/StashCollector.java

```diff
@@ -139,10 +149,40 @@ public static StashDiffReport extractDiffs(String jsonBody) throws StashReportEx
                           
                           StashDiff diff = new StashDiff(type, path, source, destination);
                           
+                          // Add comment attached to the current line
                           JSONArray jsonCommentIds = (JSONArray) jsonLine.get("commentIds");
                           if (jsonCommentIds != null) {
+                            
                             for (Object objCommentId: jsonCommentIds.toArray()) {
-                              diff.addComment((long) objCommentId);
+                              long commentId = (long) objCommentId; 
+                              

```

Comment:
later, it would be nice to split the method, very long (not required for this PR)